data/metrics_definitions.json

```diff
@@ -0,0 +1,37 @@
+[
+    {
+        "cer":
+        {
+            "short_descr": "CER calculated over the text of a whole page (in by_page) or combined text of all pages (in document_wide)",
```

Comment:
We could also need some `label` to display in the column headings.

Reply:
See d9305031de9ca2b50cf9868e7d7714fac7a63385